### PR TITLE
update parachain id

### PR DESCRIPTION
**Pull request by yooml**

### CODE REVIEW

File: packages/apps-config/src/endpoints/testingRococo.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export function createRococo (t: TFunction): LinkOption[] {
       },
       {
         info: 'rococoBifrost',
-        paraId: 107,
+        paraId: 10,
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRococo.ts:76]: Any id < 1000 is now reserved for system parachains. Not sure if this is correct.

Comment (jacogr) [reply packages/apps-config/src/endpoints/testingRococo.ts:76]: Confirmed with Shawn, it needs to be > 1000

Comment (yooml) [reply packages/apps-config/src/endpoints/testingRococo.ts]: now update to 1024

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**